docs/DRONECAN.md

```diff
@@ -18,37 +18,49 @@ In order to enable also the serial (MAVLink) communication via DroneCAN, one nee
 
 The following parameters need to be set to enable MAVLink and RC over DroneCAN.
 
-Setup the CAN driver, protocol and enable the virtual serial port, then reboot the flight controller:
+Setup the CAN driver, protocol and enable the virtual serial support, then reboot the flight controller:
 
 - CAN_P1_DRIVER = 1
-- CAN_D1_PROTOCOL = 1 (set to 1 by default)
+- CAN_D1_PROTOCOL = 1 (is set to 1 by default)
 - CAN_D1_UC_SER_EN = 1
 
-Configure the DroneCAN virtual serial port:
+Configure the DroneCAN virtual serial port (mandatory settings):
 
 - CAN_D1_UC_S1_IDX = 0
 - CAN_D1_UC_S1_NOD = 68
 - CAN_D1_UC_S1_PRO = 2
+- CAN_D1_UC_S1_BD = 57 (57600)
```

Comment:
Would suggest baud goes first here - as that would follow the order seen in the parameter list.

Reply:
done